integration_test/helper.sh

```diff
@@ -18,6 +18,7 @@ function compile_file() {
 
 function contains_all() {
     local output="$(cat -)"
+    echo "Checking if \""$*"\" are contained in \"$output\""
```

Comment:
to be removed.

Reply:
Not really, without that, if something fails in CI I never know what failed, for some reason unknown to me, all the `echo` commands below are muted, absolutely nothing is ever printed 🥲